src/CoreBundle/DependencyInjection/ChameleonSystemCoreExtension.php

```diff
@@ -211,6 +212,16 @@ private function addGoogleApiConfig(array $googleApiConfig, ContainerBuilder $co
         $definition->setArguments($args);
     }
 
+    private function addGeocoderConfig(array $geocoderConfig, ContainerBuilder $container): void
+    {
+        $definition = $container->getDefinition('chameleon_system_core.geocoding.geocoder');
+        $definition->setArgument(0, $geocoderConfig['geo_json_endpoint']);
+
+        $container->setParameter('chameleon_system_core.geocoding.attribution.show', $geocoderConfig['attribution']['show']);
+        $container->setParameter('chameleon_system_core.geocoding.attribution.name', 'test' . $geocoderConfig['attribution']['name']);
+        $container->setParameter('chameleon_system_core.geocoding.attribution.url', 'test' . $geocoderConfig['attribution']['url']);
```

Comment:
I guess the test was there for debugging purposes? :)

Reply:
`test` string has been removed